packages/haul-core/src/preset/makeConfigFactory.ts

```diff
@@ -27,6 +26,7 @@ import applySingleBundleTweaks from './utils/applySingleBundleTweaks';
 import applyMultiBundleTweaks from './utils/applyMultiBundleTweaks';
 import LooseModeWebpackPlugin from '../webpack/plugins/LooseModeWebpackPlugin';
 import InitCoreDllPlugin from '../webpack/plugins/InitCoreDllPlugin';
+import WebpackBasicBundlePlugin from '../webpack/plugins/WebpackBasicBundlePlugin';
```

Comment:
Can we come up with consistent naming?
We now have: 
- LooseModeWebpackPlugin
- WebpackBasicBundlePlugin

I think the easiest would be to go from `WebpackBasicBundlePlugin` to `BundleWebpackBasicPlugin`

Reply:
`BundleWebpackBasicPlugin` IMO doesn't make sense `BasicBundle` should go together always, so I'd suggest `BasicBundleWebpackPlugin`

Reply:
Yeah, I also noticed that, changed all occurrences to `BasicBundleWebpackPlugin`

Reply:
Lol, a typo. I meant `BasicBundleWebpackPlugin` of course